src/UniGet/RemoveTool.cs

```diff
@@ -106,8 +106,8 @@ internal static bool RemoveEmptyFolder(string path)
 
             if (ok && Directory.GetFiles(path).Any() == false)
             {
+                var metaFile = Path.GetFullPath(Path.Combine(path, "..", Path.GetFileName(path))) + ".meta";
```

Comment:
Since the directory is removed before the meta file, the path is not valid any more:
```
$ ls -l src/UniGet.Tests/bin/Release/output/Restore/Assets/UnityPackages/DepA/../DepA.meta
ls: src/UniGet.Tests/bin/Release/output/Restore/Assets/UnityPackages/DepA/../DepA.meta: No such file or directory

$ ls -l src/UniGet.Tests/bin/Release/output/Restore/Assets/UnityPackages/DepA.meta
-rw-r--r--  1 jonan  staff  106 Mar 27 23:05 src/UniGet.Tests/bin/Release/output/Restore/Assets/UnityPackages/DepA.meta
```

By calling Path.GetFullPath I was "normalizing" the path and avoiding the need of having the directory to successfully removing the meta file.